tests/browser_test.go

```diff
@@ -28,12 +28,14 @@ func TestBrowserNewPage(t *testing.T) {
 	l = len(b.Contexts())
 	assert.Equal(t, 2, l, "expected there to be 2 browser context, but found %d", l)
 
-	p.Close(nil)
+	err := p.Close(nil)
+	require.NoError(t, err)
```

Comment:
Should we reduce the number-of-lines noise?

```suggestion
	require.NoError(t, p.Close(nil))
```